src/components/Dashboard/TodoList.vue

```diff
@@ -69,7 +69,7 @@
           :class="{'invalid-section ml-n1 pl-1 rounded-0': showInvalidSection(item)}"
         >
           <div class="list-item">
-            <div class="todo-label">
+            <div class="todo-label pt-1">
```

Comment:
This (and line 2307) is a very subtle change but it better aligns the View Details button and the Continue In button.

Before:

![image](https://github.com/user-attachments/assets/af809613-03dc-477c-a6eb-b9c996756352)

After:

![image](https://github.com/user-attachments/assets/5ee0d524-114c-498c-9117-fded5c101f5a)